holidays/countries/bhutan.py

```
@@ -167,6 +171,31 @@ def _populate_subdiv_15_public_holidays(self):
+
+    @property
+    def _blessed_rainy_day_date(self) -> tuple[int, int]:
+        """Extrapolation based on available dates; the exact rule is currently unclear."""
+        dates = {
+            2012: (SEP, 22),
+            2013: (OCT, 21),
+            2015: (SEP, 22),
+        }
+        if self._year in dates:
+            return dates[self._year]
+        else:
```

Ruff (0.15.14)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)